src/cmd-artifact-disk

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3 -u
+import logging as log
+import os.path
+import sys
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, f"{cosa_dir}/cosalib")
+sys.path.insert(0, cosa_dir)
+
+from cosalib.cli import BuildCli
+import cosalib.qemuvariants as QVariants
+from cosalib.vmware import VmwareOVA
+
+# As new build targets are migrated they should be
+# defined here.
+BUILD_TARGETS = [
+    "manual",
+    "vmware",
+]
+
+
+def get_build_class(imgtype, build_root, build="latest", force=False):
+    if imgtype in QVariants.VARIANTS:
+        log.info(f"Target '{imgtype.upper()}' is a Qemu Variant image")
+        return QVariants.QemuVariantImage(build_root, build, force=force,
+                                          variant=imgtype)
+
+    if imgtype == "vmware":
+        return VmwareOVA(build_root, build, force=force)
+
+    raise Exception(f"{imgtype} is not supported by this command")
+
+
+def artifact_cli():
+    """ cli implements command-line innovation """
+    log.basicConfig(
+        format='[%(asctime)s  %(levelname)s]: %(message)s',
```

Comment:
nit: _does not need to be changed_ but generally I feel it's best to use the same quotes through a file. I'm a fan of single quote but really either works fine.